pkg/performanceprofile/controller/performanceprofile/components/machineconfig/machineconfig.go

```diff
@@ -146,8 +148,13 @@ func New(profile *performancev2.PerformanceProfile, opts *components.MachineConf
 		profile.Spec.RealTimeKernel.Enabled != nil &&
 		*profile.Spec.RealTimeKernel.Enabled
 
+	// Real time kernel with 64k-pages for aarch64 not yet supported and rejected in the validation webhook.
 	if enableRTKernel {
 		mc.Spec.KernelType = MCKernelRT
+	} else if profile.Spec.KernelPageSize != nil && *profile.Spec.KernelPageSize == performancev2.KernelPageSize("64K") {
```

Comment:
Same here about the suffix letter (K vs k). Because see that on line 41 you use 64k-pages as the kernel name (uses lowercase).